.github/runners/azure-cvm/azure-function/requirements.txt

```diff
@@ -199,26 +199,30 @@ charset-normalizer==3.1.0 \
     --hash=sha256:f8303414c7b03f794347ad062c0516cee0e15f7a612abd0ce1e25caf6ceb47df \
```

Comment:
For the sake of keeping the history clean, please don't merge the main into feature branches but just rebase it. See point 4. from our developer onboarding.